tests/e2e-pw/tests/attributes-filter/filter-products-by-attributes-count.block_theme.spec.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+import { cli } from '@woocommerce/e2e-utils';
+
+const prepareAttributes = async ( page ) => {
+	/*
+	 * Intercept the dialog event.
+	 * This is needed because when the regenerate
+	 * button is clicked, a dialog is shown.
+	 */
+	page.on( 'dialog', async ( dialog ) => {
+		await dialog.accept();
+	} );
+
+	await page.goto( '/wp-admin/admin.php?page=wc-status&tab=tools' );
+
+	await page.waitForSelector(
+		'.regenerate_product_attributes_lookup_table input'
+	);
+
+	await page.click( '.regenerate_product_attributes_lookup_table input' );
+
+	/*
+	 * Note that the two commands below are intentionally
+	 * duplicated as we need to run the cron task twice as
+	 * we need to process more than 1 batch of items.
+	 */
+	await cli(
+		`npm run wp-env run tests-cli wp action-scheduler run --hooks="woocommerce_run_product_attribute_lookup_regeneration_callback"`
+	);
+
+	await cli(
+		`npm run wp-env run tests-cli wp action-scheduler run --hooks="woocommerce_run_product_attribute_lookup_regeneration_callback"`
+	);
+};
+
+test.describe( 'Filter by Attributes Block - with All products Block', () => {
+	test.beforeAll( async () => {
+		await cli(
+			`npm run wp-env run tests-cli wp plugin activate woocommerce-blocks`
```

Comment:
This is something that I noticed and fixed in another PR #9575. We should enable the plugin on setup level (https://github.com/woocommerce/woocommerce-blocks/pull/9575/files#diff-eb99670fee72ab86b8e74c0a34e285c93b76dd5322e9629488ceefd262b33f29R14)

Reply:
@gigitux Somehow, I missed that in #9148, the WooCommerce Blocks plugin had been removed from `.wp-env.json`, see https://github.com/woocommerce/woocommerce-blocks/pull/9148/files#diff-9fd21b09a0144355cfd001e3952767df08b2e04241994470aa1d627f797b171e

Was there a particular reason not to load WooCommerce Blocks by default?

Reply:
I understand the [mappings](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/#avoid-activating-plugins-or-themes-on-the-instance) more like something that allows us to install a plugin, but not activate it by default. 

I just changed the `plugins` and `env` sections in `.wp-env.json` to this:
```json
"plugins": [
	"https://downloads.wordpress.org/plugin/woocommerce.latest-stable.zip",
	"https://github.com/WP-API/Basic-Auth/archive/master.zip",
	"https://downloads.wordpress.org/plugin/wordpress-importer.0.8.zip",
	"./tests/mocks/woo-test-helper",
	"."
],
"env": {
	"tests": {
		"mappings": {
			"wp-content/mu-plugins": "./node_modules/@wordpress/e2e-tests/mu-plugins",
			"wp-content/plugins/gutenberg-test-plugins": "./node_modules/@wordpress/e2e-tests/plugins",
			"wp-content/plugins/woocommerce-blocks": ".",
			"wp-cli.yml": "./wp-cli.yml"
		}
	}
},
```

As expected, the E2E tests of this PR passed:

<img width="1512" alt="Screenshot 2023-05-30 at 16 45 54" src="https://github.com/woocommerce/woocommerce-blocks/assets/3323310/cbd20b95-98cb-4966-819e-03da9e8a49c6">

Instead of mapping and later activating WooCommerce Blocks, I'd say it's better to activate the plugin immediately by adding it to the `plugins` section in `.wp-env.json`.

Reply:
>  I'd say it's better to activate the plugin immediately by adding it to the plugins section in .wp-env.json.

Agree! I will create a PR to fix this! Nice catch! Thanks! 💪

Reply:
Thanks for creating a separate PR for that. Feel free to assign me to review it. ;)